provisioning/miniprov/test/test_cdf.py

```diff
@@ -92,7 +95,12 @@ def test_template_sane(self):
         try:
             with open(path, 'w') as f:
                 f.write(self._get_confstore_template())
+
+            def get_machine_id() -> str:
+                return '1114a50a6bf6f9c93ebd3c49d07d3fd4'
+
             store = ConfStoreProvider(f'json://{path}')
+            store.get_machine_id = Mock(side_effect=get_machine_id)
```

Comment:
If there is no logic inside the mock, the function is not necessary:
```suggestion
            store.get_machine_id = Mock(side_effect=['1114a50a6bf6f9c93ebd3c49d07d3fd4'])
```
More details can be seen here: https://docs.python.org/3/library/unittest.mock.html#unittest.mock.Mock
Syntax is not ideal, but still can be helpful.

Reply:
Done, Above was not working as expected so I used 'return_value'